Tests/FunctionalTests/helpermethod.ts

```diff
@@ -232,3 +255,58 @@ export async function filterCheckBox(batchAtributeType: string, batchAttributeVa
   return checkBoxMatch;
  
 }
+
+export async function ExpectSpecificColumnValueDisplayed(page: Page, tablecloumnName: string, tablecloumnValue: string): Promise<void> {
```

Comment:
Please tidy up the indentation in `ExpectSpecificColumnValueDisplayed`

Reply:
Resolved indentation issue. Thanks